model/share.go

```diff
@@ -0,0 +1,26 @@
+package model
+
+import (
+	"time"
+)
+
+type Share struct {
+	ID            string    `json:"id"          orm:"column(id)"`
+	Url           string    `json:"url"`
+	Description   string    `json:"description"`
+	ExpiresAt     time.Time `json:"expires"`
+	CreatedAt     time.Time `json:"created"`
+	LastVisitedAt time.Time `json:"lastVisited"`
+	ResourceID    string    `json:"resourceID"`
```

Comment:
This needs to be annotated with `orm:"column(resource_id)"` or else it will try to match `resource_i_d` See Beego ORM documentation about [naming conventions](https://beego.me/docs/mvc/model/models.md#naming-conventions)

Reply:
Thanks for pointing that out! Corrected.